upstream/eks.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2021 The Kubernetes Authors.
```

Reply:
That's what I get for starting a feature and forgetting it for months 😄